src/Features/CSharp/Portable/Completion/KeywordRecommenders/ReadOnlyKeywordRecommender.cs

```diff
@@ -54,7 +52,7 @@ private static bool IsValidContextForType(CSharpSyntaxContext context, Cancellat
         }
 
         private static bool IsStructAccessorContext(CSharpSyntaxContext context)
-            => context.ContainingTypeDeclaration.Kind() == SyntaxKind.StructDeclaration &&
+            => context.ContainingTypeDeclaration?.Kind() == SyntaxKind.StructDeclaration &&
```

Comment:
You can use `IsKind` extension method instead. It handles null correctly and is preferred. This will be enforced whenever roslyn upgrades to a `Microsoft.CodeAnalysis.Analyzers` version that has RS1034 rule.